mapping/aspace_agent_mapping/scripts/post_agents.py

```diff
@@ -44,7 +44,7 @@ def write_name_dict_to_file(name_to_aspace_ids_map):
 
         writer = csv.writer(f)
         writer.writerow(["name", "aspace_id"])
-        writer.writerows(data)
+        writer.writerows(data.encode('utf-8'))
```

Comment:
```data``` is a list; so I don't think this will work. But this method also isn't actually used in any of the scripts -- it was used somewhere in an earlier version I think, but it's obsolete now. I've just deleted it in my version.